dags/bqetl_main_summary.py

```diff
@@ -197,6 +273,37 @@
         wait_for_copy_deduplicate_main_ping
     )
 
+    wait_for_bq_main_events = ExternalTaskSensor(
+        task_id="wait_for_bq_main_events",
+        external_dag_id="copy_deduplicate",
+        external_task_id="bq_main_events",
+        execution_delta=datetime.timedelta(seconds=3600),
+        check_existence=True,
+        mode="reschedule",
+        pool="DATA_ENG_EXTERNALTASKSENSOR",
+    )
+
+    telemetry_derived__clients_daily_event__v1.set_upstream(wait_for_bq_main_events)
+    wait_for_event_events = ExternalTaskSensor(
+        task_id="wait_for_event_events",
```

Comment:
I wonder if we should set the priority for [`event_events`](https://github.com/mozilla/telemetry-airflow/blob/ceda595853271cb71be7b4d1cacda51ebc6fe7c4/dags/copy_deduplicate.py#L65) to something >85. If Airflow, for some reason, happens to schedule `telemetry_derived__clients_daily_event__v1` and `event_events` at the same time, `telemetry_derived__clients_daily_event__v1` will have a higher priority. Likely won't make a huge difference in this single case though hopefully.

Reply:
See https://github.com/mozilla/telemetry-airflow/pull/1232